frontend/tests/e2e/apply/features/happy_path_apply.feature

```diff
@@ -0,0 +1,149 @@
+Feature: Happy Path – Apply Workflow
+  As a grant applicant
+  I want to complete the full Apply workflow successfully
+  So that I can submit a complete application without errors
+
+  Background:
+    Given the user is able to login with all roles needed
+    And the system is available
+
+  @Happy-Path
+  Scenario: Complete the Apply workflow for an Organization user
+    
+    # --- Login ---
+    Given the user is logged in
+
+    # --- Search the Opportunity ---
+    When the user searches for funding opportunity
```

Comment:
do we need to specify that the user searches for and finds an opportunity with an open competition that can be applied to?